### DISCUSSION TITLE
Default to 100% when going to the modules window

### DISCUSSION THREAD

**Discussion by alltiagocom** [DISTRHO/Cardinal#509] [Ideas]
I don't know if what I'm getting is because I have a 13" screen, but when I go to the modules page (right click, so the "search" page), it always defaults to 50%, which makes it almost impossible to see what I'm looking at, so I always have to change it to 100%. When I say "always" it's only when I add a new instance of Cardinal, not every time I go to the modules page (that would be too much). Still, having an option to set the default zoom for that page would be great

**Comment by dromer** [comment]
There is currently no way to save settings between instances.

Such settings should however be saved in your DAW project, so the next time you open the project the entire state of Cardinal is recreated including your zoom config.

**Reply by alltiagocom** [reply]
Yes, the project saves that, but what I mean is just the zoom of the modules page (right click) when we want to add a new module. Not the zoom of the patch itself. Maybe I wasn't clear on my original post? I updated it to make it clear.

**Reply by dromer** [reply]
No I understand, but it's the same issue. There is currently no way to persist settings outside of a DAW project.

**Reply by alltiagocom** [reply]
But for example for the main zoom (View > Zoom) the zoom is set to 100% by default. I wish this would be the same for the other panel. I'm not sure why the main window is set to 100%, but the search page is 50%?
